services/apps/src/apps/persistence/app_metadata.clj

```diff
@@ -710,3 +719,28 @@
                            {:u.username username})
                        {:aca.app_id (uuidify app-id)
                         :l.id       [not= (user-favorite-subselect :w.root_category_id faves-idx)]})))))
+
+(defn- list-duplicate-apps*
+  [app-name app-id category-id-set]
+  (select [:apps :a]
+          (fields :a.id :a.name :a.description)
+          (join [:app_category_app :aca] {:a.id :aca.app_id})
+          (where {(raw "trim(both from a.name)") (string/trim app-name)
```

Comment:
That's a good point about collapsing the extra whitespace down to one space, especially since multiple spaces will only be displayed as 1 space to the user in the browser.